packages/scripts/src/upgrade-tests/voting2/0_Deploy.ts

```diff
@@ -131,16 +164,18 @@ async function main() {
   console.log("Deployed VotingUpgrader: ", votingUpgrader.address);
 
   console.log("5. Deploying ProposerV2");
-
-  const defaultBond = hre.web3.utils.toWei("5000", "ether");
-
-  const proposerFactory: ProposerV2Ethers__factory = await getContractFactory("ProposerV2");
-  const proposerV2 = await proposerFactory.deploy(votingToken.address, defaultBond, governorV2.address, finder.address);
+  const proposerFactory = await getContractFactory("ProposerV2");
+  const proposerV2 = await proposerFactory.deploy(
+    votingToken.address,
+    proposerV2DefaultBond,
+    governorV2.address,
+    finder.address
+  );
   console.log("Deployed ProposerV2: ", proposerV2.address);
 
   console.log("6. Deploying EmergencyProposer");
-  const emergencyExecutor = process.env[EMERGENCY_EXECUTOR] || (await hre.ethers.getSigners())[0].address;
-  const emergencyProposerFactory: EmergencyProposerEthers__factory = await getContractFactory("EmergencyProposer");
+
+  const emergencyProposerFactory = await getContractFactory("EmergencyProposer");
```

Comment:
I noticed the emergency proposer starts with a hardcoded wait time of 1 week, but the intended value is 10 days. Are we setting it after deployment? Or planning to just change the initial hardcoded value in the contract?